config/systemd/debian-control/postrm

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+if [ -d /run/systemd/system ]; then
+    systemctl --system daemon-reload > /dev/null || true
+fi
+
+if [ "$1" = "remove" ]; then
+    if [ -x "/usr/bin/deb-systemd-helper" ]; then
+        deb-systemd-helper mask 'aktualizr.service' >/dev/null || true
+    fi
+fi
+
+if [ "$1" = "purge" ]; then
+    if [ -x "/usr/bin/deb-systemd-helper" ]; then
+        deb-systemd-helper purge 'aktualizr.service' >/dev/null || true
+        deb-systemd-helper unmask 'aktualizr.service' >/dev/null || true
```

Comment:
Is this all just to remove it if it is still running? We're no longer running it as a daemon anymore, right?

Reply:
Yes that's right. I've actually been using the network-manager debian package as a model. They use an automated debian tool to generate these steps actually but I think it would be impractical to do that through CMake/CPack which is quite rudimentary.